constellations/gemini-admin/src/account/accountInfo.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+require("./accountInfo.sass");
+import React from "react";
+import { Link } from "react-router-dom";
+
+export class AccountInfo extends React.Component {
+    formatCurrency = (amount) => {
+        // if (amount < 0) {
+        //     return "-$" + (-amount).toString();
+        // } else {
+        //     return "$" + amount;
+        // }
+        return new Intl.NumberFormat("en-US", {
+            style: "currency",
+            currency: "USD",
+        }).format(amount);
+    };
+
+    render = () => {
+        const id = this.props.id;
+        const email = this.props.email;
+        const users = this.props.users;
+        const transactions = this.props.transactions;
+        const userAddLink = "/users/" + this.props.id + "/add";
+
+        const userRows = users.map((user, index) => {
+            var name = user.firstName + " ";
+            if (user.middleName) {
+                name += user.middleName + " " + user.lastName;
+            } else {
+                name += user.lastName;
+            }
+            return (
+                <div className="row" key={index}>
+                    <span className="column">{name}</span>
+                    <span className="column">{user.email}</span>
+                    <span className="column status">
+                        {user.isGuardian
+                            ? email == user.email
+                                ? "(guardian, primary contact)"
+                                : "(guardian)"
+                            : "(student)"}
```

Comment:
Ah I see - maybe this is why I bumped into the problem before. So we COULD have the situation where a student is the primary contact. So this is two separate conditions here.
You are either a student or guardian. And if your email matches the account email, you're a primary contact.
A primary contact can be a student or guardian. But only one primary contact per account.